.gitignore

```diff
@@ -27,3 +27,9 @@ doc/api/
 *.js_
 *.js.deps
 *.js.map
+
+# AI stuff
+AGENTS.md
```

Comment:
If we're going to use AI coding assistants then it makes sense to keep hold of the context we're providing to them.

Reply:
I didn't end up having to create any prompt files

I initially added it to the gitignore out of habit, but didn't end up creating one at all